src/UserStamping.php

```diff
@@ -65,11 +72,16 @@ public static function bootUserStamping()
     }
 
     /**
-     * Add DB table columns meta info
+     * Gets columns name of model's table.
      *
+     * @return  array
      */
     protected function getUserStampingTableColumns() {
-        return \DB::connection()->getSchemaBuilder()->getColumnListing($this->getTable());
+        if ( ! $this->columns) {
```

Comment:
Please update this line by removing the whitespaces in the condition, to be consistent with the rest of the code: `if (!$this->columns) {`